config/routes.rb

```diff
@@ -6,6 +6,7 @@
   end
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root 'users#index', as: 'home'
+  
```

Comment:
- [ ] Please fix your routes to avoid errors , like this one for example 


![image](https://user-images.githubusercontent.com/75126481/218065619-983b340e-aea5-446f-bc33-747f8826807f.png)